courses/his204/quizz/052/fr.yml

```
@@ -1,4 +1,4 @@
-question: Grâce à l’intervention de qui Mirabeau échappe-t-il aux rigueurs de la répression après la publication de la Théorie de l’Impôt ?
+question: Grâce à l’intervention de qui, Mirabeau échappe-t-il aux rigueurs de la répression après la publication de la Théorie de l’Impôt ?
```

```suggestion
question: Grâce à l’intervention de qui Mirabeau échappe-t-il aux rigueurs de la répression après la publication de la Théorie de l’Impôt ?
```
Same here, no comma needed